pkg/kubelet/server/server.go

```diff
@@ -165,7 +165,9 @@ func (s *KubeletServer) Run(_ []string) error {
 	//
 	// TODO(erictune): convert all cloud provider scripts and Google Container Engine to
 	// use only --api_servers, then delete --etcd_servers flag and the resulting dead code.
-	if len(s.EtcdServerList) > 0 && len(s.APIServerList) > 0 {
+	if len(s.APIServerList) == 0 {
+		glog.Fatalf("--api_server is required.")
```

Comment:
@brendandburns Random question, isn't a standalone mode for the kubelet needed for self-bootstrapping, as well?